src/Apps/Order/Routes/Submission/index.tsx

```diff
@@ -1,24 +1,58 @@
+import { Sans, Serif } from "@artsy/palette"
 import { Submission_order } from "__generated__/Submission_order.graphql"
+import { TwoColumnLayout } from "Apps/Order/Components/TwoColumnLayout"
 import React, { Component } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import { Join } from "Styleguide/Elements/Join"
+import { Message } from "Styleguide/Elements/Message"
 import { Spacer } from "Styleguide/Elements/Spacer"
+import { Placeholder } from "Styleguide/Utils/Placeholder"
+import { Responsive } from "Utils/Responsive"
+import { SummaryFragmentContainer as Summary } from "../../Components/Summary"
 
 export interface SubmissionProps {
   order: Submission_order
+  mediator?: {
+    trigger: (action: string, config: object) => void
+  }
 }
 
 export class SubmissionRoute extends Component<SubmissionProps> {
   render() {
     const { order } = this.props
 
     return (
-      <>
-        Submission Page
-        <Spacer mb={1} />
-        {`Order #${order.id}`}
-        <Spacer mb={2} />
-        Submitted!
-      </>
+      <Responsive>
+        {() => (
+          <TwoColumnLayout
+            Content={
+              <>
+                <Join separator={<Spacer mb={3} />}>
+                  <>
+                    <Serif size="6" weight="regular" color="black100">
+                      Your order has been submitted
+                    </Serif>
+                    <Sans size="2" weight="regular" color="black60">
+                      Order #{order.code}
```

Comment:
We use the `code` (more human-readable) here instead of the ID.

Reply:
Would you update the mock in the storybook definition to give this an example value? `graphql-tools` is infering a value based off of a type, so it's rendering...

![image](https://user-images.githubusercontent.com/3087225/43926449-5018ef6a-9bf8-11e8-9af3-fb12a25249e3.png)

Good that it renders something! 😅 Would be nice to have a more relealistic value coming through though.

Reply:
https://github.com/artsy/reaction/blob/387ebe7e01dffd2bafaad83b788474cdd692e381/src/Apps/__stories__/OrderApp.story.tsx#L14

Reply:
Oh!! I actually was attempting to track down where that `"Hello World"` was coming from-- good to know it's graphql-tools. I'll add a default.